contributors/devel/contributing-api-changes.md

```diff
@@ -0,0 +1,134 @@
+# Guidelines for contributing api changes
+
+## Overview
+
+Modifications to the core Kubernetes api have a material impact on the Kubernetes project, and
+undergo thorough review and discussion before they are accepted.  This document outlines the
+review process and what is expected of api contributors.
+
+## Changes that qualify as api changes
+
+Any changes to existing or new apis are subject to the process outlined in this document.
+
+The following qualify as api changes:
+
+- Adding, removing, or modifying api types (e.g. through types.go, or by defining a new annotation)
+- Changing the version of an existing type (e.g. v1alpha1 to v1beta1)
+- Adding, removing, or modifying subresources for existing types. (e.g. /scale)
+- Material changes to how fields on existing types are interpreted / used
```

Comment:
Recommend examples here:

* A change to the default value of an API field
* A change to make validation of the API more restrictive (breaks existing clients)
* A change to make validation of the API less restrictive (may introduce security vulnerabilities, as other components depend on those fields not being able to change)
* A change to how one or more fields affect behavior of the component
* A change to constants used for a field.